guides/common/modules/proc_using-vmware-cloud-init-and-userdata-templates-for-provisioning.adoc

```diff
@@ -179,7 +180,7 @@ endif::[]
 +
 [options="nowrap" subs="+quotes"]
 ----
-# rm -f /etc/ssh/_SSH_keys_
+# rm -f /etc/ssh/\*key*
```

Comment:
```suggestion
# rm -f /etc/ssh/ssh_host_*
```
See https://www.ssh.com/academy/ssh/host-key#host-keys-in-openssh

I believe having bold text in the code block goes against our current conventions. Maybe we can instead reword the list item or explain why this is important?

Reply:
It isn't supposed to be bold text. It's supposed to be wildcard replacement in filename lookup. It's supposed to say: delete all files that contain the "key" substring.

Reply:
That's why the first asterisk is escaped.

Reply:
But your suggestion will probably work too.